server/routes/conditions/create/select-conditions/selectConditionsController.ts

```diff
@@ -17,7 +18,7 @@ export default class SelectConditionsController {
         }
       : this.populateFormFromDto(conditionsList)
 
-    const viewRenderArgs = { prisonerSummary, form: selectConditionsForm }
+    const viewRenderArgs = { prisonerSummary, form: selectConditionsForm, conditionsThatRequireNaming }
```

Comment:
This is where the controller adds the `conditionsThatRequireNaming` to the view model args, so that the nunjucks template has access to it